creator.html

```diff
@@ -152,7 +146,7 @@ <h6 class="feature-title">99.9% of Uptime</h6>
                         </div>
                         <div class="col-10">
                             <h6 class="feature-title">Live Testing</h6>
-                            <p class="grey-text">Use our <a href="https://play.google.com/store/apps/details?id=io.makeroid.companion" target="_blank">Companion</a> to see changes in real-time and how your final app will look.
+                            <p class="grey-text">Use our <a href="https://play.google.com/store/apps/details?id=io.makeroid.companion" target="_blank">Companion</a> to see changes in real-time directly on your device.  
```

Comment:
```suggestion
                            <p class="grey-text">Use our <a href="https://play.google.com/store/apps/details?id=io.makeroid.companion" target="_blank">Companion</a> to see changes in real-time directly on your device.
```

1. Remove those trailing spaces; it will cause a markdown issue.